presto-docs/src/main/sphinx/connector/snowflake.rst

```diff
@@ -0,0 +1,111 @@
+===============
+Snowflake Connector
+===============
+
+The Snowflake connector allows querying and creating tables in an external
+Snowflake database. This can be used to join data between different
+systems like Snowflake and Hive, or between two different Snowflake accounts.
+
+Configuration
+-------------
+
+To configure the Snowflake connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``snowflake.properties``, to
```

Comment:
```suggestion
in ``etc/catalog`` named, for example, ``snowflake.properties``.
```
Edit suggestion because this sentence phrasing suggests the creation of the file "mount the Snowflake connector as the ``snowflake`` catalog."